src/pages/Organization/components/OrganizationLevel.tsx

```diff
@@ -54,7 +54,7 @@ export function OrganizationLevel({
           ? t("select") +
             " " +
             t(`SYSTEM__govt_org_type__${orgType?.toLowerCase()}`)
-          : t("select_previous")
+          : t("select") + " " + t("SYSTEM__govt_org_type__district")
```

Comment:
word by word individual translations are not alloweed, the order of wordings may differ in other languages! use translation variables instead